src/ansible_navigator/actions/sample_form.py

```diff
@@ -99,9 +99,8 @@ def run(self, interaction: Interaction, app: AppPublic) -> Interaction:
         """Handle :doc
 
         :param interaction: The interaction from the user
-        :type interaction: Interaction
         :param app: The app instance
-        :type app: App
+        :return: The pending interaction
```

Comment:
This could be something like
```suggestion
        :return: The pending :class:`interaction <~Interaction>`
```
(although a full reference to the `Interaction` definition is probably necessary)

Reply:
let's see